src/canvas-view/canvas/canvas-directive.js

```diff
@@ -0,0 +1,440 @@
+(function () {
+  'use strict';
+
+  angular.module('patternfly.canvas')
+  .filter('trustAsResourceUrl', ['$sce', function ($sce) {
+    return function (val) {
+      return $sce.trustAsResourceUrl(val);
+    };
+  }])
+
+  //
+  // Directive that generates the rendered chart from the data model.
+  //
+  .directive('pfCanvas', function ($document) {
+    return {
```

Comment:
I agree, however it is not a requirement for this initial PR which requires pfCavnasView to be a component to be accessed in an Angular2 'hybrid mode'.  Converting pfCanvas to a component would allow Angular2 folks to access the basic canvas component (without toolbox & zoom) -which I'm not sure is a use case.   I've captured this in [PTNFLY-2197](https://patternfly.atlassian.net/browse/PTNFLY-2197): Remaining Issues